engine/core/IgeEngine.js

```diff
@@ -2125,8 +2125,12 @@ var IgeEngine = IgeEntity.extend({
 						if (!self.serverEmptySince) {
 							self.serverEmptySince = self.now;
 						}
-						// Kill T1 and T2 servers if it's been empty for 10+ mins.
-						if ((ige.server.tier === '1' || ige.server.tier === '2') && self.now - self.serverEmptySince > self.emptyTimeLimit) {
+						
+						const serverTier = ige.server.tier;
+						const gameTier = ige.game && ige.game.data && ige.game.data.defaultData && ige.game.data.defaultData.tier;
+						// gameTier and serverTier could be different in some cases since Tier 4 games are now being hosted on Tier 2 servers.
+						// Kill T1 and T2 servers if it's been empty for 10+ mins. Also, do not kill T2 servers if they are hosting a T4 game
+						if ((serverTier === '1' || serverTier === '2') && gameTier !== '4' && self.now - self.serverEmptySince > self.emptyTimeLimit) {
```

Comment:
I assume this new variable separates game tier from server tier, and we deploy t4 games on t2 servers?

Reply:
Yes, so the gameTier (ige.game.data.defaultData.tier) and serverTier (ige.server.tier) have already been there in existing data, I just created two new variables to avoid confusion between the two.

And yes, we are not deploying new servers to host T4 games, instead, we will use existing T2 servers only. 
The only difference is T4 Games will be hosted on all regions at once (so at any given time there will be at least 5 T2 servers (for 5 different regions) allocated to a T4 game.)